requirements.txt

```diff
@@ -2,6 +2,8 @@ argparse
 airspeed>=0.5.5
 matplotlib
 graphviz
+vispy
```

Comment:
vispy and pyqt5 are pretty heavy requirements? I had problems gettign pyqt installed on my mac. Best leave for dev reqs?